front-end/bundle-configuration/default/runtime-config.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```

Comment:
This is the application secret for the prod mode. In the `application.conf` the secret is "changeme". This shell script will override the application secret in ConductR.